frontend/leaderboard/src/Main.re

```diff
@@ -2,9 +2,8 @@ let blockDirectory =
   ([%bs.node __dirname] |> Belt.Option.getExn |> Filename.dirname)
   ++ "/src/blocks/";
 
-let clientCredentials =
-  ([%bs.node __dirname] |> Belt.Option.getExn |> Filename.dirname)
-  ++ "/../../credentials.json";
+[@bs.val]
+external credentials: string = "process.env.GOOGLE_APPLICATION_CREDENTIALS";
```

Comment:
You can probably type this as Js.Undefined.t(string) and avoid needing to do the classify below